src/retry-failed-messages/src/index.ts

```diff
@@ -1,36 +1,29 @@
 import { isError } from "shared-types"
-import getAllErrors from "./lib/getAllErrors"
+import getErrorsToRetry from "./lib/getErrorsToRetry"
 import config from "./config"
 import { AuditLogApiClient } from "shared"
-import shouldRetry from "./lib/shouldRetry"
 const apiClient = new AuditLogApiClient(config.apiUrl, config.apiKey)
 
 type OutputType = { retried: string[]; errored: string[] }
 
 export default async (): Promise<OutputType> => {
-  const messages = await getAllErrors(apiClient)
+  console.log("Fetching messages to retry")
```

Comment:
Can 'console.log's be removed from this file if we don't need them?

Reply:
They're logging so there's some visibility of that the lambda is doing